tick/optim/solver/svrg.py

```diff
@@ -51,12 +53,15 @@ class SVRG(SolverFirstOrderSto):
         Information along iteration is recorded in history each time the
         iteration number of a multiple of ``record_every``
 
-    variance_reduction : {'last', 'avg', 'rand'}, default = last
+    Other Parameters
+    ----------------
+    variance_reduction : {'last', 'avg', 'rand'}, default='last'
         Determine what is used as phase iterate for variance reduction.
 
         * 'last' : the phase iterate is the last iterate of the previous epoch
         * 'avg' : the phase iterate is the average over the iterates in the past
-          epoch
+          epoch. This is a really bad idea when using delayed updates (with
+          sparse datasets, a warning will be raised in this case)
```

Comment:
This is a really bad idea when using sparse datasets, a warning will be raised in this case

Reply:
We don't mention delayed updates anymore

Reply:
Done